templates/etc/admin/admin.conf.erb

```diff
@@ -0,0 +1,87 @@
+# The pulp server configuration
+#
+# host:
+#   The pulp server hostname
+# port:
+#   The port providing the RESTful API
+# rsa_pub:
+#   The pulp server public key used for authentication.
+# api_prefix:
+#   The REST API prefix.
+# verify_ssl:
+#   Set this to False to configure the client not to verify that the server's SSL cert is signed by
+#   a trusted authority
+#   WARNING: Setting this to False is not safe if you wish to use Pulp with real passwords or
+#            any other private information. It is strongly recommended to keep this set to True, and
+#            to use certificates that are signed by a trusted authority on the web server that
+#            serves Pulp.
+# ca_path:
+#   This is a path to a file of concatenated trusted CA certificates, or to a directory of trusted
+#   CA certificates (with openssl-style hashed symlinks, one certificate per file).
+
+[server]
+host: <%= scope['pulp::admin::host'] %>
+port: <%= scope['pulp::admin::port'] %>
+api_prefix: <%= scope['pulp::admin::api_prefix'] %>
+verify_ssl: <%= scope['pulp::admin::verify_ssl'] %>
+ca_path: <%= scope['pulp::admin::ca_path'] %>
+upload_chunk_size: <%= scope['pulp::admin::upload_chunk_size'] %>
+
```

Comment:
I think we don't need to expose everything, but if there's something generally useful... I don't know pulp well enough to say.